File: Socket/src/main/java/com/cunoc/JFlex_Cup/Token.java
package com.cunoc.JFlex_Cup;

public class Token {
    private int x;
    private int y;
    private String lexeme;

    public Token(int x, int y, String lexeme) {
        this.x = x;
        this.y = y;
        this.lexeme = lexeme;
    }

    public int getX() {
        return x;
    }

    public void setX(int x) {
        this.x = x;
    }

    public int getY() {
        return y;
    }

    public void setY(int y) {
        this.y = y;
    }

    public String getLexeme() {
        return lexeme;
    }

    public void setLexeme(String lexeme) {
        this.lexeme = lexeme;
    }

}

File: Android/app/src/main/java/com/example/boxworld_sokoban/juego/Token.kt
package com.example.boxworld_sokoban.juego

open class Token(x: Int, y: Int, lexeme: String?)  {
    private var x = 0
    private var y = 0
    private var lexeme: String? = null

    init{
        this.x = x
        this.y = y
        this.lexeme = lexeme
    }

    fun getX(): Int {
        return x
    }

    fun getY(): Int {
        return y
    }

    fun getLexeme(): String? {
        return lexeme
    }

    fun setLexeme(lexeme: String?) {
        this.lexeme = lexeme
    }

}